webhookHandler.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/go-kit/kit/log"
+	"github.com/xmidt-org/argus/chrysom"
+	"github.com/xmidt-org/argus/model"
+	"github.com/xmidt-org/webpa-common/logging"
+	"github.com/xmidt-org/webpa-common/webhook"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+type Registry struct {
+	hookStore *chrysom.Client
+	config    RegistryConfig
+}
+
+type RegistryConfig struct {
+	Logger   log.Logger
+	Listener chrysom.ListenerFunc
+	Config   chrysom.ClientConfig
+}
+
+func NewRegistry(config RegistryConfig, listener chrysom.Listener) (*Registry, error) {
+	argus, err := chrysom.CreateClient(config.Config, chrysom.WithLogger(config.Logger))
+	if err != nil {
+		return nil, err
+	}
+	if listener != nil {
+		argus.SetListener(listener)
+	}
+
+	return &Registry{
+		config:    config,
+		hookStore: argus,
+	}, nil
+}
+
+// jsonResponse is an internal convenience function to write a json response
+func jsonResponse(rw http.ResponseWriter, code int, msg string) {
+	rw.Header().Set("Content-Type", "application/json")
+	rw.WriteHeader(code)
+	rw.Write([]byte(fmt.Sprintf(`{"message":"%s"}`, msg)))
+}
+
+// get is an api call to return all the registered listeners
+func (r *Registry) GetRegistry(rw http.ResponseWriter, req *http.Request) {
```

Comment:
This is a change to the Caduceus API - previously, we were returning the registry that Caduceus has stored locally.  Now, we are returning the list of webhooks that Argus gives us, but not the list that Caduceus is currently working with.  A Get() on chrysom also doesn't even update the listener, so Caduceus could actually return a list with webhooks it doesn't have yet.